Public/manifest.json

```diff
@@ -59,6 +59,9 @@
     },
     "johnCena": {
       "description": "John Cena Time is Now"
+    },
+    "cartmanUpset": {
+      "description": "Cartman is upset"
```

Comment:
nice, i just updated the docs about not adding the keybindings in the manifest as it breaks the extension in chrome. Just the description works as well.